### DISCUSSION TITLE
Problem adding to Home Assistant

### DISCUSSION THREAD

**Discussion by Miquelsn** [Disane87/spoolman-homeassistant#169] [Q&A] [answered]
Hi,

I'm running Home Assistant in a Docker container on a Synology NAS and have successfully installed HACS and added this repository. After downloading it, I followed the instructions in the README to add the integration, but I’m encountering an issue.

When I try to add the Spoolman integration through the "Add Integration" button in Home Assistant, I cannot find it by searching for "Spoolman." I've also noticed that there’s a service related to Spoolman in the HACS integration, but I can't find a way to configure it using the Spoolman URL.

Has anyone experienced this or know how to resolve it?

Thanks for your help!

**Comment by Disane87** [comment]
Merry Christmas! Could you please describe in detail what you did to install this integration?

**Reply by Miquelsn** [reply, marked as answer]
Hi, Merry Christmas! 

Thanks for your reply.

I’m not sure why, but today when I entered Home Assistant, it mentioned there was a problem with HACS and Spoolman. It repaired itself, and now the integration appears and works as expected.

Thank you!

**Reply by Disane87** [reply]
Thank you for the fast response! Have a great evening! 🎄
